<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductOForm</class>
 <widget class="QScrollArea" name="ProductOForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <property name="alignment">
   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
  </property>
  <widget class="QWidget" name="FormContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>802</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="contentsLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>703</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Dados do Associado</string>
      </property>
      <property name="styleSheet">
       <string notr="true"> QGroupBox {
	 font-weight: bold;
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 25px;
 }

 QGroupBox::title {
	 background-color: transparent;
     subcontrol-origin: margin;
	 subcontrol-position: top left;
 }</string>
      </property>
      <property name="title">
       <string>Associado (Opcional)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnSelectAssociate">
         <property name="text">
          <string>Adicionar</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/icons.qrc">
           <normaloff>:/icons/associate_add.png</normaloff>:/icons/associate_add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnCleanAssociate">
         <property name="text">
          <string>Limpar</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/icons.qrc">
           <normaloff>:/icons/clean.png</normaloff>:/icons/clean.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QFrame" name="frameAssociate">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Nome:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblName">
            <property name="styleSheet">
             <string notr="true"> QLabel {
	 color: rgb(0, 0, 255);
 }</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Apelido:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblNickname">
            <property name="styleSheet">
             <string notr="true"> QLabel {
	 color: rgb(0, 0, 255);
 }</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>703</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Produtos do Bazar</string>
      </property>
      <property name="styleSheet">
       <string notr="true"> QGroupBox {
	 font-weight: bold;
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 25px;
 }

 QGroupBox::title {
	 background-color: transparent;
     subcontrol-origin: margin;
	 subcontrol-position: top left;
 }</string>
      </property>
      <property name="title">
       <string>Produtos do Bazar</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="0" colspan="5">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QTableWidget" name="tableProducts">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>130</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Total (R$):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTotal">
           <property name="styleSheet">
            <string notr="true"> QLabel {
	 color: rgb(0, 0, 255);
 }</string>
           </property>
           <property name="text">
            <string>0,00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" rowspan="2" colspan="4">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1" rowspan="2" colspan="5">
           <widget class="QLineEdit" name="edProductName"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>33</height>
             </size>
            </property>
            <property name="text">
             <string>Produto:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="edPrice">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>0,00</string>
            </property>
            <property name="maxLength">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>33</height>
             </size>
            </property>
            <property name="text">
             <string>Quantidade:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="2" colspan="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>33</height>
             </size>
            </property>
            <property name="text">
             <string>Preço (R$):</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QVBoxLayout" name="edQtdHolder"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>100</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QPushButton" name="btnAddProduct">
              <property name="text">
               <string>Adicionar</string>
              </property>
              <property name="icon">
               <iconset resource="../../../res/icons.qrc">
                <normaloff>:/icons/product_add.png</normaloff>:/icons/product_add.png</iconset>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCleanProducts">
              <property name="text">
               <string>Limpar</string>
              </property>
              <property name="icon">
               <iconset resource="../../../res/icons.qrc">
                <normaloff>:/icons/clean.png</normaloff>:/icons/clean.png</iconset>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="maximumSize">
       <size>
        <width>703</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> QGroupBox {
	 font-weight: bold;
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 25px;
 }

 QGroupBox::title {
	 background-color: transparent;
     subcontrol-origin: margin;
	 subcontrol-position: top left;
 }</string>
      </property>
      <property name="title">
       <string>Situação</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="rdPaid">
         <property name="text">
          <string>Quitado</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/icons.qrc">
           <normaloff>:/icons/money_received.png</normaloff>:/icons/money_received.png</iconset>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radioMoney</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdNotPaid">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>A receber</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/icons.qrc">
           <normaloff>:/icons/money_unreceived.png</normaloff>:/icons/money_unreceived.png</iconset>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radioMoney</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="maximumSize">
       <size>
        <width>703</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Observações</string>
      </property>
      <property name="styleSheet">
       <string notr="true"> QGroupBox {
	 font-weight: bold;
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 25px;
 }

 QGroupBox::title {
	 background-color: transparent;
     subcontrol-origin: margin;
	 subcontrol-position: top left;
 }</string>
      </property>
      <property name="title">
       <string>Observações</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="edObs"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnSelectAssociate</tabstop>
  <tabstop>btnCleanAssociate</tabstop>
  <tabstop>edProductName</tabstop>
  <tabstop>edPrice</tabstop>
  <tabstop>btnAddProduct</tabstop>
  <tabstop>btnCleanProducts</tabstop>
  <tabstop>tableProducts</tabstop>
  <tabstop>edObs</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="radioMoney"/>
 </buttongroups>
</ui>
